jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -164,17 +164,24 @@ ecma_op_object_get_own_property (ecma_object_t *object_p, /**< the object */
           return ECMA_PROPERTY_TYPE_NOT_FOUND_AND_STOP;
         }
 
-        ecma_number_t num = ecma_string_to_number (property_name_p);
-        ecma_string_t *num_to_str = ecma_new_ecma_string_from_number (num);
-
-        if (ecma_compare_ecma_strings (property_name_p, num_to_str))
+#ifndef CONFIG_DISABLE_ES2015_SYMBOL_BUILTIN
+        if (!ecma_prop_name_is_symbol (property_name_p))
```

Comment:
So if I understand correctly this should not be done in case of a symbol property? Symbol properties are handled somewhere else?

Would it make sense to do this?
```
if (is_symbol(..)) { break } 
...
```
this way we would not need to put macros before and after the code.

Reply:
Yeah, good idea!

Reply:
Sorry, let me answer for your question. These related code path is only for faster the way of defining/getting properties. For symbol properties we should use the "general" way.